packages/vhd-cli/src/commands/compare.js

```diff
@@ -0,0 +1,82 @@
+import { openVhd } from 'vhd-lib'
+import { getSyncedHandler } from '@xen-orchestra/fs'
+import { resolve } from 'path'
+import Disposable from 'promise-toolbox/Disposable'
+
+const compareVhd = (src, dest, path) => {
```

Comment:
My bad, I previously said to call this function `compareVhd`, but it's not what it does :grimacing: